<?xml version="1.0"?>
<launch>
<!-- This is a launch file for the race_learning_by_demonstration package-->

	<include file="$(find race_learning_by_demonstration)/launch/tabletop_segmentation_LbD.launch">
		<arg name="tabletop_segmentation_points_in" value="/camera/depth_registered/points"/> 
		<arg name="tabletop_segmentation_convert_to_base_link" default="false" />
		<arg name="tabletop_segmentation_clustering_voxel_size" default="0.001" />
		<arg name="flatten_table" default="false"/>
		<arg name="table_padding" default="0."/>
		<!--<arg name="z_filter_max" default="2.5"/>-->
	</include>

	<!--<node pkg="tf" type="static_transform_publisher" name="world_tf_broadcaster" args="0 0 0 0 0 0 /openni_rgb_optical_frame /world 500" />-->

	<!-- Launch rviz with the propper parameters for loading the atlascar rviz configuration file -->
	<node name="learning_by_demonstration" pkg="race_learning_by_demonstration" type="learning_by_demonstration" required="true" output="screen">
	</node>

	<!--<node name="gesture_detection" pkg="race_learning_by_demonstration" type="gesture_detection_node" required="true" output="screen">-->
	<!--</node>-->

</launch>
